sdk/core/azure-core-test/src/main/java/com/azure/core/test/implementation/entities/AccessPolicy.java

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.test.implementation.entities;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.OffsetDateTime;
+
+/**
+ * An Access policy.
+ */
+public class AccessPolicy {
```

Comment:
I see this duplicated in azure-core src/test as well. We should look into moving any tests in azure-core that require this class to azure-core-test later.